### PR TITLE
Upgraded NPM packages

### PR DESCRIPTION
**Pull request by johnnolan**
# Purpose

Upgrade NPM dependencies that are out of date in the service-front/web container

Fixes UML-1219

## Approach

Ran `npm-upgrade` and updated as appropriate.

## Learning

None

## Checklist

* [ ] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/web/webpack.pdf.common.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = {
             loader: 'css-loader',
             options: {
               url: true,
+              url: false,
```
Comment (cooperaj) [inline service-front/web/webpack.pdf.common.js:15]: Can't quite  decide eh?

File: service-front/web/webpack.pdf.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ module.exports = {
             loader: 'css-loader',
             options: {
               url: true,
+              url: false,
```
Comment (cooperaj) [inline service-front/web/webpack.pdf.js:22]: And again

**Review by cooperaj — CHANGES_REQUESTED** [review]
Looks good. Two tweaks

File: service-front/web/webpack.pdf.common.js (hunk 1 above)
Comment (johnnolan) [reply service-front/web/webpack.pdf.common.js:14]: lol good spot! Sorted!

**Review by cooperaj — APPROVED** [review]

**Comment by cooperaj** [comment]
oops, this is breaking local dev envs when using the dockerfile. yay

**Pull request merged**